cluster/ci/config/prow/jobs/cluster-launch-e2e.yaml

```diff
@@ -66,6 +68,79 @@ objects:
 
     containers:
 
+    # Runs any needed pre-test modifications of the installed cluster
+    - name: prepare
+      image: ${IMAGE_TESTS}
```

Comment:
what populates this variable?  it looks like the intent is to run the jenkins-e2e-test image which contains the tag-in-image.sh script but i'm not seeing how the dots are connected?

Reply:
The path starts with this part of my change:  https://github.com/openshift/release/pull/947/files#diff-118067f59af8a905b180af90c3804713R154

Setting the `as` field in the `output_image_tag_step` results in this path executing:  https://github.com/openshift/ci-operator/blob/master/pkg/steps/output_image_tag.go#L117-L134
This means the step is declaring it "provides" the template param / env var `IMAGE_TESTS` where its value is the docker pull spec for the image we produced.

Then, when you use the `ci-operator` command to instantiate the template defined in this file, this code executes to complete the loop:  https://github.com/openshift/ci-operator/blob/master/pkg/steps/template.go#L50-L79